manifest_staging/charts/csi-secrets-store-provider-azure/values.yaml

```diff
@@ -161,6 +161,9 @@ secrets-store-csi-driver:
   syncSecret:
     enabled: false
 
+  tokenRequests:
```

Comment:
Where are we using this in template?

Reply:
It's a driver property: https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/manifest_staging/charts/secrets-store-csi-driver/values.yaml#L211-L215